cloneable-modules/users/Controllers/Admin/UsersController.php

```diff
@@ -12,6 +13,59 @@ class UsersController extends AdminApiController
      */
     protected $controllerInfo = [
         'repository' => 'users',
-        'listOptions' => [],
+        'listOptions' => [
+            'select' => ['id','name','user_group_id','email'],
+            'filterBy' => [],
+            'paginate' => null, // if set null, it will be automated based on repository configuration option
+        ],
+        'rules' => [
+            'all' => [
```

Comment:
Set it empty to allow the developer to determine what to validate by him/herself.